src/Driver/VersionResolver.php

```diff
@@ -11,6 +11,8 @@
 
 interface VersionResolver
 {
+    public const CHROME_MAJOR_VERSION_ENDPOINT_BREAKPOINT = 115;
```

Comment:
This const does not belong in the generic driver interface, but in the chrome specific one

Reply:
Yep true